src/Pages/Category.tsx

```diff
@@ -5,12 +5,13 @@ import { SpecialBanner } from "../Components/SpecialBanner";
 import TawkTo from "../Components/TawkTo";
 import React, { useState, useEffect } from 'react';
 import { SliderPlaceholder } from "../Components/Card/SliderPlaceholder";
+import env from "./../env.json"
 
 export function Category() {
     let [categories, setCategories] = useState<any[]>([]);
     let arr:number[] = [1,2,3,4];
     useEffect(() => {
-        fetch(process.env.REACT_APP_API_BASE+"/categories")
```

Comment:
Was using process.env not working for you? It has been working for me and Casey. We are using .env.local files to hold our environment variables.